packages/react/src/patterns/sections/CardSectionImages/CardSectionImages.js

```diff
@@ -21,7 +21,8 @@ const CardSectionImages = ({ cards, ...otherProps }) => {
     ({ image, eyebrow, heading, copy, cta: { href } }) =>
       image && eyebrow && heading && !copy && href
   );
-  return <CardSection {...otherProps} cards={cardsWithImages} />;
+  const { ['cta']: _, ...newObj } = otherProps;
```

Comment:
This is written out in a confusing way. Is there anyreason why we couldn't just use `delete`?

```
delete otherPropse.cta;
return <CardSection {...otherProps} cards={cardsWithImages} />;
```

Reply:
I also thought the delete operator, but I may get any comments from you on that delete operator , so tried another syntax to remove specific prop from spread operator.